src/printer/OITimeToolUserOutputBuilder.cc

```diff
@@ -0,0 +1,22 @@
+#include "OITimeToolUserOutputBuilder.h"
+#include "common/Exception.h"
+
+#include <sstream>
+
+namespace s2j {
+namespace printer {
+
+const std::string OITimeToolUserOutputBuilder::FORMAT_NAME = "oittuser";
+
+std::string OITimeToolUserOutputBuilder::dump() const {
+    std::stringstream ss;
+    ss << "__RESULT__ " << encodeStatusCode() << " " << userMilliSecondsElapsed_
```

Comment:
For some reason the upstream sioworkers use the default oitt format instead of oiaug, while not utilizing the encoded status code.
I will switch this to oiaug-like format and sometime soon create a PR in sioworkers to use oiaug, as it does indeed allow for simpler code.

Reply:
> the upstream sioworkers

uh...  um... yeah last time I checked there was quite a bit of differences between upstream sioworkers and prod sioworkers....

Reply:
I personally find it funny that there probably does not exist a production deployment with the actual upstream version.